### ISSUE TITLE
Mouse pointer location differs from selected text

### ISSUE THREAD

**Issue by byt3m4st3r** [raphamorim/rio#573]
Rarely, but sometimes, a mouse is helpful to copy some commands or text, but if the mouse is used to select text in the Rio terminal, the selection divers from the mouse pointer location.

```txt
  Machine     >  MacBookPro18,4
  Kernel      >  23.5.0
  OS          >  macOS 14.5.0 Unknown
  DE          >  Aqua
  WM          >  Quartz Compositor
  Terminal    >  rio (Version 0.1.2)
  Shell       >  zsh
  CPU         >  Apple M1 Max (10)
  Resolution  >  3024x1964@120fps (as 1512x982)
                 6880x2880@60fps (as 3440x1440)
                 1920x1080
```

![rio-mouse-test](https://github.com/user-attachments/assets/c9c27644-c3ad-4a22-ab48-342631c88021)

**Comment by MrPandir** [comment]
Duplicate https://github.com/raphamorim/rio/issues/535

**Comment by byt3m4st3r** [comment]
The issue occurs with this navigation config:

```txt
[navigation]
mode = "CollapsedTab"
clickable = false
```

**Comment by raphamorim** [comment]
Good catch with this issue! I fixed in the main branch, will be released in v0.1.3. Tomorrow should be a nightly version available for testing. Let me know anything, closing it for now
